src/LibraryViewExtensionMSWebBrowser/web/library/library.html

```diff
@@ -341,17 +361,20 @@
             if (searchBar == null)
                 return;
 
-            var cutoffArea = document.getElementById("cutoff_area");
-            var height = (searchBar.clientHeight + tools.clientHeight);
-
             //Apply a specific style for the <div> depending if the parameter enable is true or false.
             if (enable == true) {
-                cutoffArea.style.display = "block";
-                cutoffArea.style.top = height + "px";
-                cutoffArea.style.height = addons.clientHeight + "px";
+                tools.classList.add("overlay");
+                searchBar.classList.add("overlay");
+                let children = addons.childNodes;
```

Comment:
I know that I'm part of this, but it would be better if we check the nulls and array sizes. 😅

Reply:
@filipeotero  a validation for null and array size was added in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12312/commits/2933fbf31bc87989ba49d35636b97c904c986d5f